sw-src.js

```diff
@@ -0,0 +1,6 @@
+importScripts('https://unpkg.com/workbox-sw@1.3.0');
+
+const VERSION = 1;
```

Comment:
yeah that's a good idea, since this is linked to the docs, are there any vars that are updated whenever the docs are rebuilt?